sympy/physics/vector/vector.py

```diff
@@ -522,6 +522,8 @@ def diff(self, var, frame, var_in_dcm=True):
         >>> N = ReferenceFrame('N')
         >>> A = N.orientnew('A', 'Axis', [q1, N.y])
         >>> A.x.diff(t, N)
+        - sin(q1)*q1'*N.x - cos(q1)*q1'*N.z
+        >>> A.x.diff(t, N).express(A)
         - q1'*A.z
```

Comment:
So for very simple cases like this, the expressions will be longer, because the old old internal call to express will automatically simplify things (because SymPy does some really basic simplification automatically), but for any non-trivial example, the expressions are significantly longer with the old way.